src/classes/Account.js

```diff
@@ -254,29 +260,34 @@ class Account {
 
 	migrate = () => (
 		new Promise((resolve) => {
+			ghosteryDebug.addAccountEvent('migrate', 'migrate start');
 			const legacyLoginInfoKey = 'login_info';
 			chrome.storage.local.get(legacyLoginInfoKey, (items) => {
 				if (chrome.runtime.lastError) {
+					ghosteryDebug.addAccountEvent('migrate', 'runtime error');
```

Comment:
@wlycdgr This throws an error that `addAccountEvent` does not exist

Reply:
Addressed